www/css/cytube.css

```diff
@@ -266,16 +238,24 @@ li.ui-sortable-helper, li.ui-sortable-placeholder + li.queue_entry {
 
 .server-msg-disconnect {
     border: 1px solid #ff0000;
-    margin: 5px;
-    padding: 5px;
+    padding-top: 5px;
+    padding-bottom: 5px;
+    margin-top: 5px;
+    margin-right: 5px;
     color: #ff0000;
+    text-align: center;
```

Comment:
This CSS class is also used for kick/ban messages as well as the message that you can't chat because you're muted.  Have you checked to see whether `text-align: center` makes those look bad?